### PR TITLE
feat: add logout functionality to employeehome

### PR DESCRIPTION
**Pull request by pk97**

### CODE REVIEW

**Comment by pk97** [comment]
image

**Comment by pk97** [comment]
image

**Comment by pk97** [comment]
image

File: src/components/Avatar.js [hunk 1]
```diff
@@ -1,12 +1,76 @@
...
+export default function Avatar({imageLink, logout, email, name}) {
+  const anchorRef = useRef(null);
+  const [open, setOpen] = useState(false);
+
+  const handleToggle = () => {
+    setOpen(!open);
+  };
+
+  const handleClose = (event) => {
+    if (anchorRef.current && anchorRef.current.contains(event.target)) {
```
Comment (apoorv-j) [inline src/components/Avatar.js]: Can we avoid this by wrapping MUIAvatar as well in ClickAwayListener? If not, then it is fine

File: src/components/Avatar.js [hunk 2]
```diff
@@ -1,12 +1,76 @@
...
+        imgProps={{referrerpolicy: "no-referrer"}}
+      />
+      <ClickAwayListener onClickAway={handleClose}>
+      <Popper open={open} anchorEl={anchorRef.current}>
+        <Box className="popper">
+        <Typography className={"name"}> {name}</Typography>
+          <Typography
+            sx={{
+              color: "rgba(0, 0, 0, 0.60)",
+              fontFamily: "Roboto",
```
Comment (apoorv-j) [inline src/components/Avatar.js]: As discussed in the standup today, we should be having Nunito font across the website. There is some discrepancy in the figma file

File: src/components/Avatar.js [hunk 3]
```diff
@@ -1,12 +1,76 @@
...
+            {email}
+          </Typography>
+          <Divider />
+          <Button
+            onClick={() => {
+              logout();
+            }}
+           style={{
+             color: "rgba(0, 0, 0, 0.60)",
+             fontFamily: 'Nunito',
```
Comment (apoorv-j) [inline src/components/Avatar.js]: Check if we need to give this style again here as Nunito is defined as the default font at the Provider level

File: src/pages/home/Employee/EmployeeHome.js [hunk 4]
```diff
@@ -89,7 +89,7 @@ function EmployeeHome() {
                 marginLeft: "auto"
               }}
          >
-           <Avatar imageLink={user?.picture} />
+           <Avatar imageLink={user?.picture} logout={logout} email={user?.email} name={user?.name} />
```
Comment (apoorv-j) [inline src/pages/home/Employee/EmployeeHome.js]: We can rename the Avatar component as AvatarWithPopper

File: src/components/Avatar.js (hunk 2 above)
Comment (pk97) [reply src/components/Avatar.js]: Done

File: src/components/Avatar.js (hunk 3 above)
Comment (pk97) [reply src/components/Avatar.js]: Done

File: src/pages/home/Employee/EmployeeHome.js (hunk 4 above)
Comment (pk97) [reply src/pages/home/Employee/EmployeeHome.js]: Done

File: src/App.js [hunk 5]
```diff
@@ -16,7 +16,7 @@ function App() {
         <Route path="/" element={<Login />} />
         <Route
           path="/home"
-          element={userRole === "employee" ? <EmployeeHome /> : <AdminHome />}
+          element={userRole === "Employee" ? <EmployeeHome /> : <AdminHome />}
```
Comment (mishra-s) [inline src/App.js]: We are using employee only, I checked it on auth dashboard.

**Comment by mishra-s** [comment]
Can we fix this?

Screenshot 2023-07-19 at 7 11 06 AM

**Comment by pk97** [comment]
Pushed a fix

> Can we fix this?
> 
> Screenshot 2023-07-19 at 7 11 06 AM

Done

**Review by apoorv-j — APPROVED** [review]

**Review by apoorv-j — APPROVED** [review]

**Pull request merged**